package.json

```diff
@@ -18,6 +18,8 @@
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
     "react-hot-loader": "^3",
+    "react-router-dom": "^5.1.2",
+    "react-router-hash-link": "^1.2.2",
```

Comment:
I think using hashes is a good idea for a site like this (it's easier to deploy than regular paths).